README.md

```diff
@@ -1,61 +1,122 @@
 
 # YYJ Tech Code of Conduct
 
+YYJ Tech strives to be as equitible and inclusive as possible. This code of conduct sets the expectations and guidelines for all members on how to conduct themselves to create a community that is as welcoming and inclusive as possible, as well as the consequences for unacceptable behavior.
+
+However, we are all humans, and we know that there are times that we will not get things right. 
+
+Help us out by sharing things you are concerned about here. You can find more information about how to do that in the [Reporting Guidelines](https://github.com/yyjtech/code-of-conduct/blob/master/reporting_guidelines.md).
```

Comment:
I read "things you are concerned about here" as if it were a link (ie like "click [here](https://some.place.where.i.go.to.give.feedback.com) to share things you are concerned about".  Maybe this could be worded as:

> Help us out by sharing things you are concerned about on YYJ Tech.   You can find more information about how to do that in the [Reporting Guidelines](https://github.com/yyjtech/code-of-conduct/blob/master/reporting_guidelines.md)

with the rest